pkg/server/commands/list_objects.go

```diff
@@ -379,7 +390,9 @@ func (q *ListObjectsQuery) ExecuteStreamed(
 		defer cancel()
 	}
 
-	err := q.evaluate(timeoutCtx, req, resultsChan, maxResults)
+	resolutionMetadata := connectedobjects.NewResolutionMetadata()
+
+	err := q.evaluate(timeoutCtx, req, resultsChan, maxResults, resolutionMetadata)
```

Comment:
why don't we send the resolution metadata inside the `if !channelOpen` block?

Reply:
Added for StreamedListObjects